src/Relay/Connection/Paginator.php

```diff
@@ -53,7 +55,7 @@ public function backward(ArgumentInterface $args, $total, array $callableArgs =
     }
 
     /**
-     * @return ConnectionInterface|object A connection or a promise
```

Comment:
see previous

Reply:
Tested this method with Xdebug and fixed to Connection|Promise